packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -1,11 +1,26 @@
-@import '../../../global/global.pcss';
-@import '../../../global/mixins.pcss';
+@import '../../../../global/global.pcss';
+@import '../../../../global/mixins.pcss';
 @import './styles/generated-text.pcss';
 @import './styles/feedback-buttons.pcss';
 @import './styles/rephrase-buttons.pcss';
 @import './styles/typing-indicator.pcss';
 @import './styles/source-citations.pcss';
 
+[part='generated-text'] {
```

Comment:
this is already included in the import `@import './styles/generated-text.pcss';`